test/Cake.MinVer.Tests/Cake.MinVer.Tests.csproj

```diff
@@ -8,8 +8,8 @@
   <ItemGroup>
     <PackageReference Include="Cake.Common" Version="1.0.0" />
     <PackageReference Include="Cake.Testing" Version="1.0.0" />
-
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" />
+    <PackageReference Include="Verify.Xunit" Version="11.0.0-beta.9" />
```

Comment:
Any interesting reason to use `11.0.0-beta.9` prerelease or would I be fine with stable `10.9.1`?

Reply:
you would be. i will be deploying a stable of 11 in the next 24hours. leave this open and i will move it to 11

Reply:
Awesome! Cheers mate!